app/src/test/java/com/codesoom/assignment/controllers/TaskControllerMVCTest.java

```diff
@@ -0,0 +1,124 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.TaskService;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.hamcrest.Matchers.containsString;;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.BDDMockito.given;
+
+@AutoConfigureMockMvc
+@SpringBootTest
+@DisplayName("TaskController 테스트")
+class TaskControllerMVCTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @MockBean
+    private TaskService taskService;
+
+    private final String TASK_TITLE = "test";
+    private List<Task> tasks;
+    private Task task;
+
+    @BeforeEach
+    void setUp() {
+        tasks = new ArrayList<>();
+
+        task = new Task();
+        task.setId(1L);
+        task.setTitle(TASK_TITLE);
+    }
+
+    @Nested
+    @DisplayName("GET /tasks가 호출되면")
```

Comment:
http 테스트니까 요청이라는 표현이 더 적절할 거에요.
```suggestion
    @DisplayName("GET /tasks가 요청되면")
```